cpp/dolfinx/geometry/BoundingBoxTree.h

```diff
@@ -71,7 +71,13 @@ class BoundingBoxTree
   /// Return bounding box coordinates for a given node in the tree
   /// @param[in] node The bounding box node index
   /// @return A copy of bounding box (lower_corner, upper_corner). Shape (2,3).
-  std::array<double, 6> get_bbox(std::size_t node) const;
+  std::array<double, 6> copy_bbox(std::size_t node) const;
```

Comment:
I don't think this is need and it's redundant. The caller can copy the span into and array if they want a copy.